### ISSUE TITLE
Unknown layer type : registry.count(type) == 1 (0 vs. 1) when trainning

### ISSUE THREAD

**Issue by daisysnow** [akirafukui/vqa-mcb#17]
I met registry.count(type) == 1 (0 vs. 1)  when training.
I google it and someone told me to change this flag WITH_PYTHON_LAYER := 1 in Makefile.config
I remake caffe but it seems the issue reappeares.
Since I'm not familiar with Caffe, is there any suggestions @akirafukui 
Thanks a lot!

Issues are shown like :
Check failed: registry.count(type) == 1 (0 vs. 1) Unknown layer type: SoftAttention (known types: AbsVal, Accuracy, ArgMax, BNLL, BatchNorm, BatchReindex, Bias, CompactBilinear, Concat, ContrastiveLoss, Convolution, Crop, Data, Deconvolution, Dropout, DummyData, ELU, Eltwise, Embed, EuclideanLoss, Exp, Filter, Flatten, HDF5Data, HDF5Output, HingeLoss, Im2col, ImageData, InfogainLoss, InnerProduct, Input, L2Normalize, LRN, LSTM, LSTMUnit, Log, MVN, MemoryData, MultinomialLogisticLoss, PReLU, Parameter, Pooling, Power, Python, RNN, ReLU, Reduction, Reshape, SPP, Scale, Sigmoid, SigmoidCrossEntropyLoss, SignedSqrt, Silence, Slice, Softmax, SoftmaxWithLoss, Split, TanH, Threshold, Tile, WindowData)
*** Check failure stack trace: ***
Aborted (core dumped)

**Comment by daisysnow** [comment]
I found it was solved  in #15
